pwndbg/typeinfo.py

```diff
@@ -45,8 +45,8 @@ def lookup_types(*types):
 def update():
 
     module.char   = gdb.lookup_type('char')
-    module.ulong  = lookup_types('unsigned long', 'uint', 'u32', 'uint32')
-    module.long   = lookup_types('long', 'int', 'i32', 'int32')
```

Comment:
why do we remove those? those help us work with Rust or Go targets iirc

Reply:
I don't think ulong type is always uint32. And that's the reason why sometimes some kernel pointers get casted to 32bit and raise errors

Reply:
Seems that we should determine the lookup type based on language (`show language`) :/

Reply:
Fwiw the best way to handle it would be to add support to gdb so we could create `gdb.Type` instances out of thin air...